src/bot/integrations/discord.ts

```diff
@@ -303,30 +307,40 @@ class Discord extends Integration {
-    if (channel) {
-      const message = await (channel as DiscordJs.TextChannel).messages.fetch(this.embedMessageId);
-      const embed = message?.embeds[0];
-      if (message && embed) {
-        embed.setColor(0xff0000);
-        embed.setDescription(`${oauth.generalChannel.charAt(0).toUpperCase() + oauth.generalChannel.slice(1)} is not streaming anymore! Check it next time!`);
-        embed.spliceFields(0, embed.fields.length);
-        embed.addFields([
-          { name: prepare('webpanel.responses.variable.game'), value: api.stats.currentGame},
-          { name: prepare('webpanel.responses.variable.title'), value: api.stats.currentTitle},
-          { name: prepare('integrations.discord.streamed-at'), value: `${this.embedStartedAt} - ${moment().tz(timezone).format('LLL')}`, inline: true},
-          { name: prepare('webpanel.views'), value: api.stats.currentViews, inline: true},
-          { name: prepare('webpanel.followers'), value: api.stats.currentFollowers, inline: true},
-        ]);
-        embed.setImage(`https://static-cdn.jtvnw.net/ttv-static/404_preview-1920x1080.jpg?${Date.now()}`);
-
-        if (oauth.broadcasterType !== '') {
-          embed.addField(prepare('webpanel.subscribers'), api.stats.currentSubscribers, true);
+    try {
+      const channel = this.client?.guilds.cache.get(this.guild)?.channels.cache.get(this.sendOnlineAnnounceToChannel);
+      if (channel) {
+        const message = await (channel as DiscordJs.TextChannel).messages.fetch(this.embedMessageId);
+        const embed = message?.embeds[0];
+        if (message && embed) {
+          embed.setColor(0xff0000);
+          embed.setDescription(`${oauth.generalChannel.charAt(0).toUpperCase() + oauth.generalChannel.slice(1)} is not streaming anymore! Check it next time!`);
+          embed.spliceFields(0, embed.fields.length);
+          embed.addFields([
+            { name: prepare('webpanel.responses.variable.game'), value: api.stats.currentGame},
+            { name: prepare('webpanel.responses.variable.title'), value: api.stats.currentTitle},
+            { name: prepare('integrations.discord.streamed-at'), value: `${this.embedStartedAt} - ${moment().tz(timezone).format('LLL')}`, inline: true},
+            { name: prepare('webpanel.views'), value: api.stats.currentViews, inline: true},
+            { name: prepare('webpanel.followers'), value: api.stats.currentFollowers, inline: true},
+          ]);
+          embed.setImage(`https://static-cdn.jtvnw.net/ttv-static/404_preview-1920x1080.jpg?${Date.now()}`);
+  
+          if (oauth.broadcasterType !== '') {
+            embed.addField(prepare('webpanel.subscribers'), api.stats.currentSubscribers, true);
+          }
+          message.edit(embed);
         }
-        message.edit(embed);
       }
+      this.embedMessageId = '';
+    } catch (e) {
+      warning(e.stack);
+    }
+
+    try {
```

Comment:
I am still not sure why it is in both try/catch, you already have try/catch in changeClientOnlinePresence function, just move it to line 306